transaction/account/type_simple_transfer.go

```diff
@@ -0,0 +1,57 @@
+package account
+
+import (
+	"github.com/Concordium/concordium-go-sdk"
+)
+
+type SimpleTransferResultEvent struct {
+	*concordium.TransactionResultEvent `json:""`
+	From                               *concordium.Address `json:"from"`
+	Amount                             *concordium.Amount  `json:"amount"`
+	To                                 *concordium.Address `json:"to"`
+}
+
+type SimpleTransferRejectReason struct {
+	*concordium.TransactionRejectReason `json:""`
+
+	// TODO
+	// Array of different types? Really???!!!
+	// [
```

Comment:
I'm not sure this belongs here.

Reply:
And it is a representation of a tuple, that is why the types in the array are different.